libnetwork/drivers/bridge/setup_ip_tables_linux_test.go

```diff
@@ -499,3 +482,87 @@ func TestMirroredWSL2Workaround(t *testing.T) {
 		})
 	}
 }
+
+func setupWSL2MirroredMode(t *testing.T, loopback0 bool, wslinfoPerm os.FileMode) func() {
+	if loopback0 {
+		iface := &netlink.Dummy{
+			LinkAttrs: netlink.LinkAttrs{
+				Name: "loopback0",
+			},
+		}
+		err := netlink.LinkAdd(iface)
+		assert.NilError(t, err)
```

Comment:
Is this something that needs cleaning up afterwards, or not a problem if we don't?

Reply:
Both times this function is called a disposable network namespace is created first. I'll add a comment stating that it should be used that way.